### PR TITLE
Adding size and colour to Avatar component

### PR DESCRIPTION
**Pull request by gjones**
### Summary
Adding the option of choosing between `md` and `sm` for the size of the `Avatar`. Also Adding different options for colours. Starting with 3, can add more easily later. 

https://github.com/user-attachments/assets/bdab443d-8b2f-4509-a815-b3950c8ff540

### CODE REVIEW

**Comment by rndD** [comment]
What if we change the name of the component to something more generalized? 
`CircularContainer`, something like that.

**Comment by rndD** [comment]
Also additional use cases: 
image

For it we need to more colors

**Comment by gjones** [comment]
I think that is extending Click Icon to have a `backgroundColour` prop.

File: src/components/Avatar/Avatar.tsx [hunk 1]
```diff
@@ -6,17 +6,34 @@ import {
 } from "@radix-ui/react-avatar";
 import styled from "styled-components";
 
+type Size = "md" | "sm";
 type TextSize = "md" | "sm";
+type State = "neutral" | "success" | "vivid";
```
Comment (vineethasok) [inline src/components/Avatar/Avatar.tsx:11]: should we add info and danger and warning to these
I think neutral can be changed to default

Comment (vineethasok) [inline src/components/Avatar/Avatar.tsx:11]: Where are we using vivid state?

Comment (vineethasok) [inline src/components/Avatar/Avatar.tsx:11]: May be we can also add custom state which inherits the parent components background and color

Comment (rndD) [reply src/components/Avatar/Avatar.tsx:11]: The notification bell, for example, when it shows unread notifications.

**Comment by rndD** [comment]
> I think that is extending Click Icon to have a `backgroundColour` prop.

What about a rounded shape? Should it be a custom or added to `<Icon />` too?

**Comment by gjones** [comment]
I'll close this as the changes I made are not consistent with what an avatar actually is.

**Pull request closed without merging**